ghostdr/ghost/primitives_ghost_slit.py

```diff
@@ -255,7 +255,8 @@ def processSlits(self, adinputs=None, **params):
                 earliest_end = min(sc_end, sv_end)
                 overlap = (earliest_end - latest_start).seconds
                 overlap = 0.0 if overlap < 0.0 else overlap  # no overlap edge case
-                sv_duration = (sv_end - sv_start).seconds
+                #sv_duration = (sv_end - sv_start).seconds
+                sv_duration = ext.hdr['EXPTIME']
```

Comment:
Also, can we guarantee that the EXPTIME will always come back in seconds, to the accuracy we require?

Reply:
The descriptor should be fine if that's the preferred way to access the header information.  EXPTIME looks to be written in as seconds and is recorded at fractions of a second.  I suggested this change since the sv_end and sv_start appear to only be recorded to the level of accuracy of a whole second, so short (< 1s) exposures will make sv_duration = 0 and cause divide by 0 errors later.  I'm not sure if this would cause problems if exposures are stopped:  I'm not sure if EXPTIME is the original requested exposure time or if it is the net exposure time, so it's possible this might not be the best solution either.

Reply:
Ohhh I'd forgotten the point about the EXPTIME possibly being the requested time rather than the actual value, that may have been why it was done that way in the first place now that you remind me.... who can we ping on the instrument control side to check that?

Reply:
I'm not sure, I'll email Alan McConnachie and see if he can point us to the right person to ask.